modules/cas_cache/layer_cache_management.c

```diff
@@ -2353,13 +2353,30 @@ int cache_mngt_activate(struct ocf_mngt_cache_standby_activate_config *cfg,
 	if (result == -KCAS_ERR_WAITING_INTERRUPTED)
 		goto out_cache_put;
 
+	result = _cache_start_finalize(cache, -1, true);
```

Comment:
Before calling finalize you have to make sure that `ocf_mngt_cache_standby_activate()` finished successfully